packages/components/chip/src/chip.ts

```diff
@@ -1,45 +1,18 @@
-import { buildProps } from '@puik/utils'
-import type { PuikTooltipPosition } from '@puik/components'
-import type { ExtractPropTypes, PropType } from 'vue'
-import type Chip from './chip.vue'
+import type { PuikTooltipPositions } from '@prestashopcorp/puik-components';
+import type Chip from './chip.vue';
 
-export const chipSizeVariants = ['default', 'small'] as const
+export enum PuikChipSizeVariants {
+  Default = 'default',
+  Small = 'small',
+}
 
-export type PuikChipSizeVariant = (typeof chipSizeVariants)[number]
+export interface ChipProps {
+  id: string
+  content: string
+  size?: `${PuikChipSizeVariants}`
+  icon?: string
+  disabled?: boolean
+  tooltipPosition?: `${PuikTooltipPositions}`
```

Comment:
`` PuikTooltipPositions | `${PuikTooltipPositions}` ``

Reply:
Nope `${PuikTooltipPositions}` allows passing the enum or the values of the enum

Reply:
Same comment that above.